README.md

```diff
@@ -24,6 +41,60 @@ If you'd like to contribute, you'll probably want to get Sorbet up and running b
 1. Use VSCode and install `watchman` (`brew install watchman` on macOS) & the Sorbet extension
 1. Add a pre-commit hook that runs `srb tc`
 
+### Adding a new utility
+
+If you'd like to add a new utility, please...:
+
+1. Add a new command in `lib/command` with the following structure:
+
+```ruby
+# typed: strict
+
+class Command::NewCommand < Command
+  extend T::Sig
+
+  sig {params(options: Options, config: Config, client: TumblrClient).void}
+  def call(options, config, client)
+    Base::IterateThroughPosts.call(options, config, client) do |post|
+      puts "Updating post #{post.id} (#{post.post_url})" if options.verbose
+      # do something
+    end
+  end
+end
+```
+
+1. Add the command to the `Command::Command` enum in `lib/lib/command.rb`:
+
+```diff
+ class Command < T::Enum
+   enums do
+     PrivatizePosts = new
+     UpdateCommunityLabels = new
++    NewCommand = new
+   end
+ end
+```
+
+1. Add the new file to `lib/tumblr_utilties.rb` to ensure it's loaded:
+
+```ruby
+# note, order matters!
+require_relative('command/new_command.rb')
+```
+
+1. Add the new command call to the entry `script.rb` file:
+
+```ruby
+when Command::Command::NewCommand
+  Command::NewCommand.call(options, config, client)
+```
```

Comment:
if we end up adding a lot more commands, we can get rid of this structure but I appreciate the safety from the enum & manual inclusion of files :)